### ISSUE TITLE
Fix Admin UI Accessibility Violations

### ISSUE THREAD

**Issue by agagancarczyk** [keycloak/keycloak#26058]
### Before reporting an issue

- [X] I have read and understood the above terms for submitting issues, and I understand that my issue may be closed without action if I do not follow them.

### Area

admin/ui

### Describe the bug

Review serious and critical violations recorded in the cypress dashboard and provide fixes.

### Version

22.0.5

### Expected behavior

Cypress dashboard should not show any serious and critical a11y violations.

### Actual behavior

Cypress dashboard shows a couple of serious and critical a11y violations.

### How to Reproduce?

Take a look at the cypress dashboard/ accessibility section.Cypress
